### ISSUE TITLE
Parser error in version 4.3.0

### ISSUE THREAD

**Issue by marek-parfianowicz** [openclover/clover#73]
When attempting to instrument using the maven plugin, or when building through IntelliJ I'm receiving this parse error.

Caused by: line 8:39: expecting STAR, found 'module' at clover.antlr.Parser.match(Parser.java:211) at com.atlassian.clover.instr.java.JavaRecognizer.identifierStar(JavaRecognizer.java:1004) at com.atlassian.clover.instr.java.JavaRecognizer.importDefinition(JavaRecognizer.java:945) at com.atlassian.clover.instr.java.JavaRecognizer.compilationUnit(JavaRecognizer.java:847) at com.atlassian.clover.instr.java.Instrumenter.instrument(Instrumenter.java:211) at com.atlassian.clover.instr.java.Instrumenter.instrument(Instrumenter.java:124)

The offending source is

import com.fasterxml.jackson.databind.module.SimpleModule;

If I comment out this import and usages of SimpleModule, the instrumentation on the project completes as expected. This occurs only after upgrading to 4.3.0, 4.2.1 still works as expected.

**Comment by marek-parfianowicz** [comment]
Original report: https://bitbucket.org/openclover/clover/issues/73

**Comment by marek-parfianowicz** [comment]
It's a duplicate of #72.
